src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -470,12 +478,13 @@ function ( $matches ) {
 	 * @since 6.5.0
 	 *
 	 * @param WP_Interactivity_API_Directives_Processor $p               The directives processor instance.
+	 * @param string                                    $mode            Whether the processing is entering or exiting the tag.
 	 * @param array                                     $context_stack   The reference to the context stack.
 	 * @param array                                     $namespace_stack The reference to the store namespace stack.
 	 */
-	private function data_wp_interactive_processor( WP_Interactivity_API_Directives_Processor $p, array &$context_stack, array &$namespace_stack ) {
+	private function data_wp_interactive_processor( WP_Interactivity_API_Directives_Processor $p, $mode, array &$context_stack, array &$namespace_stack ) {
```

Comment:
Nit, to follow with other params, it could get annotated as a string here and in other places:
```suggestion
	private function data_wp_interactive_processor( WP_Interactivity_API_Directives_Processor $p, string $mode, array &$context_stack, array &$namespace_stack ) {
```

Reply:
Changed in [this commit](https://github.com/WordPress/wordpress-develop/pull/6267/commits/8c90eced3693ef746334e7bc018ed08db5afc80a).